regex-assembly/932370.ra

```diff
@@ -4,7 +4,7 @@
 ##! Word list for rule 932370 (RCE Windows command injection part 1/2)
 ##!
 ##! The list comes from the project LOLBAS. You can get it using the following one-liner:
-##! `curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/LOLBAS-Project/LOLBAS/git/trees/master\?recursive\=1 | jq -r '.tree[].path ' | grep ^yml/ | cut -f3 -d/ | cut -f1 -d. | tr 'A-Z' 'a-z' | sort | uniq`
+##! `curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/LOLBAS-Project/LOLBAS/git/trees/master | jq -r '.tree[].path ' | grep ^yml/ | cut -f3 -d/ | cut -f1 -d. | tr 'A-Z' 'a-z' | sort | uniq`
```

Comment:
Is the recursive not needed? 🤔

Reply:
Yes! I tried again. My first test had a flaw 😅

Fixed in [c725699](https://github.com/coreruleset/coreruleset/pull/3232/commits/c725699d2bb4c7f4e7d968a9eab8e4099d304add)